app/views/docs/index.phtml

```diff
@@ -13,7 +13,7 @@ $cols = [
     'xxxl' => ['nav' => 'span-2', 'main' => 'span-10'],
 ]
 ?>
-<div class="zone <?php echo $layout; ?> docs" data-general-strech-value="<?php echo $layout; ?>">
+<div class="zone <?php echo $layout; ?> docs" data-general-stretch-value="<?php echo $layout; ?>">
```

Comment:
```suggestion
<div class="zone <?php echo $layout; ?> docs" data-general-strech-value="<?php echo $layout; ?>">
```

We should keep the typo here until this is also fixed in appwrite/appwrite css files and other templates.